lib/libzrepl/data_conn.c

```diff
@@ -740,6 +757,9 @@ uzfs_zvol_rebuild_dw_replica(void *arg)
 
 			if (start_rebuild_from_clone == 1) {
 				start_rebuild_from_clone = 2;
+				wait_snap_timeout();
+				uzfs_zvol_set_rebuild_status(zinfo->main_zv,
+				    ZVOL_REBUILDING_AFS);
 				rc = uzfs_zinfo_rebuild_from_clone(zinfo);
```

Comment:
We have to quiesce IOs after changing state to AFS state.

Reply:
good catch. done.